frontend/fingertips-frontend/.env.development

```diff
@@ -1 +1,2 @@
 FINGERTIPS_API_URL="http://localhost:5144"
+APPLICATIONINSIGHTS_CONNECTION_STRING=
```

Comment:
Is it worth adding something to the README about how to configure this locally (this file is for when running the Next dev server only). Is that something we'd want to support? If not, I'd suggest we don't include this here.

Reply:
You may want to test it / work on it but not as a matter of course.  I've removed it for now - people can always add it back in.